tests/test_pcovr.py

```diff
@@ -46,7 +46,7 @@ def test_simple_prediction(self):
         for space in ["feature", "sample", "auto"]:
             with self.subTest(space=space):
                 pcovr = self.model(
-                    mixing=0.0, n_components=self.X.shape[-1], space=space
+                    mixing=0.0, n_components=1, space=space
```

Comment:
I just want to comment on this change: when `mixing` is set to 0.0, only `n_properties` nonzero components can exist. This particular test was failing on me because additional components with tiny eigenvalues (slightly bigger than `tol`) were being propagated through the model and introducing numerical precision issues. It is probably a good idea to put more robust constraints on the number of singular values in the case of `mixing = 0`.